api/schemas/LocationInfo.yaml

```diff
@@ -0,0 +1,33 @@
+# {
+#  "timestamp-ms": 1585906080346,
+#  "userId": 34753543,
+#  "anonymous":false,
+#  "inside": true,
+#  "placeIds": [
+#    453785436,
+#    71984147938
+#  ]
+# }
+type: object
+description: Specifics of an user's current location relative to a place.
+required:
+  - timestampMs
+  - userId
+  - anonymous
+  - inside
+  - placeIds
+# TODO add properties description and examples?
+properties:
+  timestampMs:
+    type: integer
+    format: uint64
```

Comment:
Perchè qua il timestamp è in millisecondi, mentre in AppConnection è una stringa in formato date-time?

Reply:
mmm non mi ero accorto che è stato fatto diversamente negli altri posti. è una preferenza mia nel senso che usare le timestamp elimina la complessità delle timezone che sono presenti nelle date-time ("....+0000") con tutto il casino che deriva da ora solare/legale